### DISCUSSION TITLE
education benefits hasn't been received

### DISCUSSION THREAD

**Discussion by DJjjjhao** [community/community#139048] [GitHub Education]
### Hi everyone,

I have been approved for the educational benefits, but it did not appear after three days

**Comment by davevad93** [comment]
Hi @DJjjjhao ,

What's the status of your application?

If the label is green and states "`Congrats! Your request for @DJjjjhao  was approved and your benefits will be available soon.` that means you have to wait. It usually takes a few days, but sometimes it can take a little longer, even a couple of weeks; the waiting time may vary depending on different factors, such as the volume of requests.

If the label is purple and states "`Congrats! Your request for @DJjjjhao  was approved and your benefits are now available.`", that means you should have access to your benefits. Try first going to the [pack offers page](https://education.github.com/pack) and select one of the available offers, if you can access them you are fine, if not I suggest you to raise a ticket [here](https://support.github.com/contact/education), select "`I am a verified student having trouble redeeming my Global Campus offers`" and explain your issue.

For further reference, take a look at the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352):

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)

And also the following discussions: #132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721

**Reply by DJjjjhao** [reply]
Dear Davide,
I found my status was accepted just now, thank you very much!

**Reply by davevad93** [reply]
Hey @DJjjjhao , you're welcome, have a nice day! 👍

**Reply by DJjjjhao** [reply]
Thank you! Have a nice day!
